lib/i18n/ar.js

```diff
@@ -173,7 +173,9 @@ export class Arabic extends AbstractLanguage {
           }
 
           // Process every number other than 2
+          /** @todo Figure out why 2 is so special */
```

Comment:
Because in Arabic there is: singular, dual, and plural starts at 3.
https://en.wikipedia.org/wiki/Dual_(grammatical_number)

Reply:
@forzagreen That makes perfect sense, thank you very much!